core/client/kt/verify_test.go

```diff
@@ -51,6 +51,7 @@ func mustMetadataAsAny(t *testing.T, meta *keytransparency_v1_types.MapperMetada
 // Test vectors were obtained by observing the integration tests, in particular by adding logging
 // output around the calls to GetEntry and VerifyGetEntryResponse in grpc_client.go, and the input
 // to merkle.VerifyMapInclusionProof in VerifyGetEntryResponse.
+//    fmt.Printf("%# v", pretty.Formatter(e))
```

Comment:
remove this commented out line?

Reply:
I thrashed around trying to remember this line, so I thought I'd include it for posterity.

Reply:
ok, fair enough.